<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Infeitel GNSS Viewer V4.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalls">
    <property name="title">
     <string>设备</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="actionConfiguration_View"/>
    <addaction name="actionBrowse"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbcd"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="actionMap"/>
    <addaction name="actionRate"/>
    <addaction name="actionTime"/>
    <addaction name="actionSatPosion"/>
    <addaction name="actionSatData"/>
    <addaction name="actionSatLevel"/>
   </widget>
   <widget class="QMenu" name="menuConsole">
    <property name="title">
     <string>输出信息</string>
    </property>
    <addaction name="actionBinaryConsole"/>
    <addaction name="actionTextConsole"/>
   </widget>
   <addaction name="menuCalls"/>
   <addaction name="menuConsole"/>
   <addaction name="menuTools"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionConfigure"/>
   <addaction name="actionBrowse"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
   <property name="toolTip">
    <string>About program</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
   <property name="toolTip">
    <string>连接设备</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>断开</string>
   </property>
   <property name="toolTip">
    <string>断开设备</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>设置串口</string>
   </property>
   <property name="toolTip">
    <string>设置串口</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionBrowse">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/browser.png</normaloff>:/images/browser.png</iconset>
   </property>
   <property name="text">
    <string>打开日志文件目录</string>
   </property>
   <property name="toolTip">
    <string>浏览日志文件</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出程序</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionConfiguration_View">
   <property name="text">
    <string>配置设备</string>
   </property>
  </action>
  <action name="actionMap">
   <property name="text">
    <string>世界地图</string>
   </property>
  </action>
  <action name="actionRate">
   <property name="text">
    <string>速度表</string>
   </property>
  </action>
  <action name="actionTime">
   <property name="text">
    <string>时间</string>
   </property>
  </action>
  <action name="actionSatPosion">
   <property name="text">
    <string>卫星位置</string>
   </property>
  </action>
  <action name="actionSatData">
   <property name="text">
    <string>卫星信息</string>
   </property>
  </action>
  <action name="actionSatLevel">
   <property name="text">
    <string>卫星信号值</string>
   </property>
  </action>
  <action name="actionBinaryConsole">
   <property name="text">
    <string>Binary窗口</string>
   </property>
   <property name="toolTip">
    <string>调试信息窗口</string>
   </property>
  </action>
  <action name="actionTextConsole">
   <property name="text">
    <string>Nmea窗口</string>
   </property>
   <property name="toolTip">
    <string>Nmea信息窗口</string>
   </property>
  </action>
  <action name="actionAbcd">
   <property name="text">
    <string>使用说明</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="terminal.qrc"/>
 </resources>
 <connections/>
</ui>
